back-end/controllers/login.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const router = express.Router();
+const { queryDB } = require('../helpers/db-helpers');
+
+// Existing user
+router.post('/login', [
+    check('email').exists().isEmail().normalizeEmail(),
+    check('password').exists()
+], async function (req, res, next){
+    try {
+        const { email, password } = req.body;
+
+        const query = `SELECT password, id FROM Users WHERE email = ${email}`;
+        const results = await queryDB(query);
+    
+        if(!dbResults) {
+            return res.status(404).send('Not found.');
```

Comment:
Here we can use `401` instead of 404. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401

Reply:
Good call, changed